teams_rules.go

```diff
@@ -101,6 +107,28 @@ type TeamsCheckSessionSettings struct {
 	Duration Duration `json:"duration"`
 }
 
+type (
+	DnsResolverSettings struct {
```

Comment:
can we please prefix these types (like the others) to avoid polluting the global namespace?